app/build.gradle

```diff
@@ -1,13 +1,13 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.2"
+    compileSdkVersion 25
+    buildToolsVersion '25.0.0'
 
     defaultConfig {
         applicationId "org.loklak.android.wok"
-        minSdkVersion 11
-        targetSdkVersion 23
+        minSdkVersion 14
```

Comment:
Maybe due to incomplete build it was asking to use minSdk to be 14, but I again retried and it works. Will change it.